### ISSUE TITLE
Primary eMail address not respected

### ISSUE THREAD

**Issue by exapplerep** [zanonmark/Google-4-TbSync#29]
Consider a contact named John Smith with three eMail addresses entered in this order in the following example:
ccc@mac.com
aaa@yahoo.com
bbb@yahoo.com

Assume there are no other contacts. When crafting a new outgoing eMail, TBird will suggest the aaa@yahoo.com address as the primary even if you type in "ccc" or type in "john". In other words: The primary eMail setting isn't being respected.

See this [TBird bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1750462). I created the bug report but now realize the problem is with this addon (I think...).

**Comment by zanonmark** [comment]
Hi,

thanks for Your report. However I couldn't reproduce it.

1) I created the (only one) contact in the addressbook:
![1](https://user-images.githubusercontent.com/13437344/150395342-40c01d6d-fd9d-4aa0-bc10-53809e3e8a87.png)

2) I synchronized the TB addressbook:
![2](https://user-images.githubusercontent.com/13437344/150395392-45998f33-4d27-4b7f-b2a9-ad5b53826303.png)

3) When querying for Smith, all I see is the 1st (ccc@mac.com) and 2nd (aaa@yahoo.com) email addresses:
![3](https://user-images.githubusercontent.com/13437344/150395465-c30bd83e-bd16-4832-b3e9-1e61ca6b5e90.png)

Did You assign any specific label to the email addresses on the Google side?

Thanks,
MZ
